src/Symfony/Bundle/FrameworkBundle/Console/Application.php

```diff
@@ -55,6 +56,14 @@ public function getKernel()
         return $this->kernel;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function reset()
```

Comment:
what about also adding it to the base Application class (with empty implementation) so that IDEs/static analyzer don't complain `->getApplication()->reset()` doesn't exist?